ApiDoctor.Validation/OData/ODataParser.cs

```diff
@@ -59,6 +60,7 @@ public class ODataParser
             { "Edm.SByte", -12 },
             { "Edm.DateTime", "2014-01-01T00:00:00Z" },
             { "Edm.DateTimeOffset", "2014-01-01T00:00:00Z" },
+            { "Edm.Duration", "P3DT1H" },
```

Comment:
Please provide a more complete example (including minutes and seconds in this case)